### ISSUE TITLE
Problem with the number of extracted features samples

### ISSUE THREAD

**Issue by FlorencDemrozi** [fraunhoferportugal/tsfel#32]
Dear authors, 
First of all, congratulations on this great project very helpful fo all the community. 
I have a issue related to the number of extracted features samples:
I execute this call
X = ts.time_series_features_extractor(cfg, tmp_data, fs = 32, window_size=32, overlap=0, verbose = 0)

On my accelerometer data frame of dimension 160 x 3, 160 samples and three columns ['X','Y','Z'].
From this call, X has a dimension of 1 x 789. It returns a single sample of features for all the 160 x 3 accelerometer samples.
However, this does not seem right. Since window _size = 32 (1 second of time frame), it has to return to me an X whit dimension 5 x 789. 
How is this possible.

**Comment by lmfernandes96** [comment]
Dear @FlorencDemrozi,

Thank you for your issue and for using TSFEL.

The problem that you present seems to be related to the fact that you are not setting the window_spliter parameter to True.
This is actually an interesting observation, and since it may induce the user to errors, we will work in a new version that does not require setting the windows_spliter parameter to True.

If the problem persists, please feel free to contact us again and even sent us your accelerometer data so that we can test it with more detail.

**Comment by FlorencDemrozi** [comment]
Thank you for the prompt response.
I also tried by using the window_spliter = True, it works correctly, however, in some input data instances it doesn't work and returns the same error as previously. 
I don't see what can be the course. 

```
> /Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/tsfel/feature_extraction/features.py:685: RuntimeWarning: divide by zero encountered in true_divide
> 
>   return (p_end - p_init) / (x_end - x_init)
> 
> Traceback (most recent call last):
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/features_extraction.py", line 37, in <module>
> 
>     overlap=0)
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/tsfel/feature_extraction/calc_features.py", line 176, in time_series_features_extractor
> 
>     features = calc_window_features(dict_features, wind_sig, fs, features_path=features_path)
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/tsfel/feature_extraction/calc_features.py", line 289, in calc_window_features
> 
>     eval_result = eval(execf, locals())
> 
>   File "<string>", line 1, in <module>
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/tsfel/feature_extraction/features.py", line 715, in ecdf_percentile
> 
>     return tuple(np.repeat(signal[0], len(percentile)))
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/pandas/core/series.py", line 871, in __getitem__
> 
>     result = self.index.get_value(self, key)
> 
>   File "/Users/cristianturetta/PycharmProjects/PD-FoG/ml-venv/lib/python3.7/site-packages/pandas/core/indexes/base.py", line 4404, in get_value
> 
>     return self._engine.get_value(s, k, tz=getattr(series.dtype, "tz", None))
> 
>   File "pandas/_libs/index.pyx", line 80, in pandas._libs.index.IndexEngine.get_value
> 
>   File "pandas/_libs/index.pyx", line 90, in pandas._libs.index.IndexEngine.get_value
> 
>   File "pandas/_libs/index.pyx", line 138, in pandas._libs.index.IndexEngine.get_loc
> 
>   File "pandas/_libs/hashtable_class_helper.pxi", line 998, in pandas._libs.hashtable.Int64HashTable.get_item
> 
>   File "pandas/_libs/hashtable_class_helper.pxi", line 1005, in pandas._libs.hashtable.Int64HashTable.get_item
> 
> KeyError: 0
```

**Comment by lmfernandes96** [comment]
Dear @FlorencDemrozi,

I will kindly ask you to send me some of your data files that are having this issue.
Hopefully, I can help you faster.

Thank you

**Comment by FlorencDemrozi** [comment]
Here it is,
Thank you.
In case, my email: florenc.demrozi@univr.it

[fdem_code.tar.gz](https://github.com/fraunhoferportugal/tsfel/files/4623249/fdem_code.tar.gz)

**Comment by lmfernandes96** [comment]
Dear @FlorencDemrozi,

I believe that we found the source of this issue, it has already been corrected on the development version. I will kindly ask you to try the development version of TSFEL.

You should uninstall the TSFEL version that you are currently working on and then install the development version by typing:
pip install https://github.com/fraunhoferportugal/tsfel/archive/v0.1.3.tar.gz

If more issues arise, feel free to contact us.

**Comment by FlorencDemrozi** [comment]
This works as I need. 
Thank you very much.
Once again, great job.

**Comment by CristianTuretta** [comment]
Dear authors, 
the solution proposed by @lmfernandes96 works. However, in my setup I have noticed that `from IPython import get_ipython` is missing in file `tafel/feature_extraction/calc_features.py`.
Once added this line of code everything seems to work fine.

My setup is:
-  OS: macOS 10.15.4
-  Interpreter: Python 3.7
-  IDE: PyCharm 2019.3.5

**Comment by lmfernandes96** [comment]
Dear @CristianTuretta,

Thank you for your interest in TSFEL.

This bug is already corrected on the development version. Soon we will make a new release with these bugs corrected.

If you detect any other problem, feel free to contact us.

**Comment by lmfernandes96** [comment]
Issue solved in **Release version 0.1.3**
